### ISSUE TITLE
Are pull requests for the original Electron.NET planned?

### ISSUE THREAD

**Issue by GregorBiswanger** [theolivenbaum/electron-sharp#12]
Hello @theolivenbaum 

when we accepted your last pull requests back then, there were some bugs that made Electron.NET no longer executable. We have requested several times to fix this. Unfortunately, we haven't heard from you again, which has severely delayed the general further development of Electron.NET. My question now is whether there is anything else coming from you? Otherwise I currently have to undo all the pull requests, where there is a lot of work behind it and unfortunately your great work so far will be lost. It would be nice if we could progress together as a community. Please give me feedback so I know where we are.

Thanks.

**Comment by r-pankevicius** [comment]
We could merge as a community, what I'd strongly suggest. However "original Electron.NET" was failing too much to release updates that @theolivenbaum had a free time to make not one but even two forks of it.

So what's your motivation to keep the "original Electron.NET" moving forward?

**Comment by GregorBiswanger** [comment]
The update delay came about because we were waiting for updates from @theolivenbaum. I have found a new co-developer who is also interested in continuing to work intensively on the project. So far I was alone on the project and I had children at home, which limited my time very much. I have now tried to get an answer from @theolivenbaum in different ways. e-mail, under Pull Requests/Issues and now here. We'll probably reset the whole git history and continue with our new strategy of our own. Unfortunately, that's not how it works.

**Comment by r-pankevicius** [comment]
I see. Maybe you should invite @theolivenbaum as a contributor if he wants to have one repo and shows willingness to contribute. Because having several repos each with their own bugs if a bad idea.

**Comment by theolivenbaum** [comment]
Hi @GregorBiswanger, 

As you're know, I've been maintaining a fork of Electron.NET for a while under a temporary name - the renaming was only to clean this up on our end. 

I would be happy to merge things back to Electron.NET, but like you, I don't have the bandwidth to handle rebasing the code, testing and validating all the changes that have been merged on the original repo since the fork. There were many issues we fixed over the last year - deadlocks, cross-platform problems, updating to a newer SocketIO library, etc - and at this stage it would be a big risk for us to go back to an untested codebase.

I'm open for any ideas on how to handle this moving forward considering the constraints on both sides.
